pleskdistup/main.py

```diff
@@ -356,13 +357,20 @@ def main():
         "-v", "--version", action="store_true",
         help="show the version of this utility."
     )
+    parser.add_argument("--locale", default="en_US.UTF-8", help="set the locale for the utility")
 
     options, extra_args = parser.parse_known_args()
     if getattr(options, "help", False):
         parser.print_help()
     else:
         options.help = False
 
+    # Configure locale to avoid problems on systems where LANG or LC_CTYPE changed,
+    # while files on the system still has utf-8 encoding
+    # We should do it before initializing logger to make sure utf-8 symbols will be
+    # written correctly to the log file.
+    locale.setlocale(locale.LC_CTYPE, options.locale)
```

Comment:
I think, we shouldn't call `setlocale()` if `--locale` isn't set, otherwise we would be always overriding user system settings.

Reply:
Actually I can't see any reason to have `--local` then, we could just ask a customer simply use environment variables.